Character.cpp

```diff
@@ -22,7 +24,39 @@ void Character::attack(Character&  target) const{
 bool Character::isDead() const{
 	return getHp()==0;
 }
+Character Character::parseUnit(const std::string &filename){
+	std::ifstream file;
+    file.open(filename);
+	if (!file.is_open()) throw -99;
```

Comment:
Kodindentalas mar megint, vagy meg mindig.

Reply:
Bocsi, VSCode megviccelt 2x is